tests/functional/dispersion/dispersion.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python3
+
+import pyphare.pharein as ph #lgtm [py/import-and-import-from]
+from pyphare.pharein import Simulation
+from pyphare.pharein import MaxwellianFluidModel
+from pyphare.pharein import ElectromagDiagnostics, FluidDiagnostics
+from pyphare.pharein import ElectronModel
+from pyphare.simulator.simulator import Simulator
+from pyphare.pharein import global_vars as gv
+
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import numpy as np
+mpl.use('Agg')
+
+
+
+def fromNoise():
+
+    # in this configuration there are no prescribed waves
+    # and only eigen modes existing in the simulation noise
+    # will be visible
+
+    Simulation(
+        smallest_patch_size=20,
+        largest_patch_size=20,
+
+        # the following time step number
+        # and final time mean that the
+        # smallest frequency will be 2/100
+        # and the largest 2/dt  = 2e3
+        time_step_nbr=100000,
+        final_time=100.,
+
+        boundary_types="periodic",
+
+        # smallest wavelength will be 2*0.2=0.4
+        # and largest 50
+        cells=500,
+        dl=0.2,
+        diag_options={"format": "phareh5",
+                      "options": {"dir": "dispersion",
+                                  "mode":"overwrite"}}
+    )
+
+    def density(x):
+        return 1.
+
+
+    def by(x):
+        return 0.
+
+
+    def bz(x):
+        return 0.
+
+
+    def bx(x):
+        return 1.
+
+
+    def vx(x):
+        return 0.
+
+
+    def vy(x):
+        return 0.
+
+    def vz(x):
+        return 0.
+
+
+    def vthx(x):
+        return 0.01
+
+
+    def vthy(x):
+        return 0.01
+
+
+    def vthz(x):
+        return 0.01
+
+
+    vvv = {
+        "vbulkx": vx, "vbulky": vy, "vbulkz": vz,
+        "vthx": vthx, "vthy": vthy, "vthz": vthz
+    }
+
+    MaxwellianFluidModel(
+        bx=bx, by=by, bz=bz,
+        protons={"charge": 1, "density": density, **vvv}
+    )
+
+    ElectronModel(closure="isothermal", Te=0.12)
```

Comment:
is this `Te` the most appropriate for the wave to see ?

Reply:
at the time these were ran the pressure term was commented anyway, I'll change to Te=0